mozilla/blurts-server
# MNTOR-978: Migrate breach resolution (part 2)

# References: 
Jira: MNTOR-978
Figma: n/a




# Description
Migrating breach resolution from v1 to v2. This one-off script changes all recency indices to breach IDs 


# Screenshot (if applicable)

Not applicable.

# How to test
- Resolve a few breaches in v1
- Run the syncBreaches.js script to make sure breaches are present in your db
- Run the deprecateBreachResolve script (part 1) to translate those breaches from v1 to v2
- Run this script to see the recency indices changed to breach IDs that match the breach table


# Checklist (Definition of Done)
